doc/contributing/technical-priorities.md

```diff
@@ -35,42 +36,16 @@ to enterprises that are considering expanding their preferred platforms to
 include Node.js. It is, therefore, important that the Node.js project work
 to ensure there are good typings available for the public Node.js APIs.
 
-## Documentation
-
-The current documentation is great for experienced developers or people
-who are aware of what they are looking for. On the other hand, for
-beginners this documentation can be quite hard to read and finding the
-desired information is difficult. We must have documentation
-that is suitable for beginners to continue the rapid growth in use.
-This documentation should include more concrete examples and a learning
-path for newcomers.
+## Serverless
 
-## WebAssembly
+Serverless is a cloud computing model where the cloud provider manages the underlying infrastructure and automatically allocates resources as needed. Developers only need to focus on writing code for specific functions, which are executed as individual units of work in response to events. Node.js is one of the main technology used by developers in this field therefore it is crucial for us to provide a great solution.
```

Comment:
Ultimately most serverless runtimes are closed-source forks of Node.js, patched to support the needs of their cloud. Given that Serverless vendors are not keen to participate in the development of Node.js, nor do they provide public feedback, I don't see how this priority could be translated into actual work.

I'm +1 to support whoever shows up and wants to collaborate with the Node.js project. Supporting companies that are not willing to give back is not something I can endorse.

I think a goal could be to have those companies start collaborating on the project, _but_ I don't think "serverless" could be a technical priority.